File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/MobilWarisan.java
package id.ac.uph.pbo.pertemuan01;

public class MobilWarisan extends Kendaraan {
    private int jumlahPintu;

    public MobilWarisan(String merek, int jumlahPintu) {
        super(merek);   // memanggil constructor Kendaraan
        this.jumlahPintu = jumlahPintu;
    }

    // Method tambahan khusus Mobil
    public void bukaKap() {
        System.out.println("[Mobil:" + merek + "] Membuka kap mesin.");
    }
}

File: PYTHON/Pertemuan 01/MobilWarisan.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
from Kendaraan import Kendaraan

class MobilWarisan(Kendaraan):
    def __init__(self, merek: str, jumlah_pintu: int):
        super().__init__(merek)
        self.jumlah_pintu = jumlah_pintu

    def buka_kap(self):
        print(f"[Mobil:{self.merek}] Membuka kap mesin.")

